packages/delegation-process/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "pagopa-interop-delegation-process",
+  "version": "1.0.0",
+  "description": "PagoPA Interoperability service for delegations management",
+  "main": "dist",
+  "type": "module",
+  "scripts": {
+    "test": "vitest",
+    "test:it": "vitest integration",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node --loader ts-node/esm -r 'dotenv-flow/config' --watch ./src/index.ts",
```

```suggestion
    "start": "tsx -r 'dotenv-flow/config' --watch ./src/index.ts",
```
In main we dropped ts-node in favor of tsx